integration/kubernetes/confidential/agent_image_encrypted.bats

```diff
@@ -7,6 +7,7 @@
 load "${BATS_TEST_DIRNAME}/lib.sh"
 load "${BATS_TEST_DIRNAME}/../../confidential/lib.sh"
```

Comment:
`tests_common.sh` loads these two files, so they should be available without the explicit load